### PR TITLE
Docs: Private registry tutorial

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Adding tutorial on accessing container images from a private registry.

### CODE REVIEW

File: docs/tutorials/private-registries.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+```
+
+Lastly, we can deploy the operator inside our cluster with referencing our new values.yaml manifest to override the default values:
+
+```
+helm upgrade --install trivy-operator aqua/trivy-operator \
+  --namespace trivy-system \
+  --create-namespace \
+  --version 0.1.9
+  --values ./values.yaml
```
Comment (chen-keinan) [inline docs/tutorials/private-registries.md:50]: also can be achieved in this way
```suggestion
helm upgrade --install trivy-operator aqua/trivy-operator \
  --namespace trivy-system \
  --create-namespace \
  --version 0.1.9
  --set="trivy.command=fs"
```

**Comment by chen-keinan** [comment]
@AnaisUrlichs I love it , added small nit comment , not a must.

**Review by chen-keinan — APPROVED** [review]

File: docs/tutorials/private-registries.md (hunk 1 above)
Comment (AnaisUrlichs) [reply docs/tutorials/private-registries.md:50]: @chen-keinan but you would also have to give the operator root access at the same time, no?

```
helm upgrade --install trivy-operator aqua/trivy-operator \
  --namespace trivy-system \
  --create-namespace \
  --version 0.1.9
  --set="trivy.command=fs"
  --set="trivyOperator.scanJobPodTemplateContainerSecurityContext.runAsUser=0"
```

**Comment by AnaisUrlichs** [comment]
@chen-keinan updated :)

**Comment by chen-keinan** [comment]
LGTM 🚀

**Pull request merged**